tests/test_regression_krige.py

```diff
@@ -62,11 +58,19 @@ def test_regression_krige():
 @pytest.mark.skipif(not SKLEARN_INSTALLED,
                     reason="requires scikit-learn")
 def test_krige_housing():
+    import ssl
+    import urllib
+
     try:
         housing = fetch_california_housing()
-    except PermissionError:
-        # This can raise permission error on Appveyor
-        pytest.skip('Failed to load california housing dataset')
+    except (ssl.SSLError, urllib.error.URLError):
+        ssl._create_default_https_context = ssl._create_unverified_context
```

Comment:
This looks suspicious to me, it shouldn't be necessary.

Reply:
This is to prevent an error on travis for macOS. There an SSLError pops up and this test fails.
See: https://travis-ci.com/github/GeoStat-Framework/PyKrige/jobs/302762299#L703

Reply:
Should I just skip it on macos?

Reply:
I will keep it, so we still test this on macOS. I will keep an eye on it, if this ssl situation changes in the future on Travis.